sdk/core/core/CMakeLists.txt

```diff
@@ -7,9 +7,22 @@ project (az_core LANGUAGES C)
 
 set(CMAKE_C_STANDARD 99)
 
-add_library (az_core src/az_core.c)
+add_library (az_core src/az_core.c src/az_json_read.c)
+
+if(MSVC)
+  target_compile_options(az_core PRIVATE /W4 /WX)
```

Comment:
/WX and -Werror need to be behind a toggle. Otherwise packagers will need to modify the cmakelists.txt file to make things work with future compilers that introduce new warnings.

Also adding /W4 here by itself may cause the build to fail when combined with /WX on cmake versions < 3.15 (because /W3 is already on the command line)

Reply:
We need to merge PR #35

Reply:
I don't see the relevant change from that pr?

Reply:
Issue #41